amplipi/ctrl.py

```diff
@@ -482,6 +500,15 @@ def _update_src_info(self, src: models.Source):
     else:
       src.info = models.SourceInfo(img_url='static/imgs/disconnected.png', name='None', state='stopped')
 
+  def _get_source_config(self, sources: Optional[List[models.Source]]=None) -> List[bool]:
+    """ Convert the preamp's source configuration """
+    if not sources:
+      sources = self.status.sources
+    src_cfg = [True] * 4
+    for s, src in enumerate(sources):
+      src_cfg[s] = self._is_digital(sources[s].input)
```

Comment:
probably should be `src` instead of `sources[s]`

Reply:
It is limited to physical sources. I don't see a future where we have more sources than physical sources though.